src/test/fixtures/profiles/make-profile.js

```diff
@@ -490,3 +490,29 @@ function _buildThreadFromTextOnlyStacks(
   });
   return thread;
 }
+
+function getNetworkMarker(startTime: number, id) {
+  const payload: NetworkPayload = {
+    type: 'Network',
+    id,
+    pri: 0,
+    status: 'DONE',
```

Comment:
The status `DONE` got me confused as I thought we just have `START` and `STOP`, so I thought I rather check in if there is something I am missing? :)

Reply:
It was just mock data, I didn't remember what the strings were off the top of my head, since they aren't in our types. I'll change it to `STOP` to be consistent and not confusing.